runtime/doc/pi_health.txt

```diff
@@ -3,16 +3,64 @@
 Author: TJ DeVries <devries.timothyj@gmail.com>
 
 ==============================================================================
-1. Usage
+1. Contents					 *health.vim-contents*
 
-The health checker system is a way to get a snapshot of the important
-information in a system.
+	1. Contents ..: |health.vim-contents|
+	2. Health.vim introduction ..: |health.vim-intro|
+	3. Health.vim manual ..: |health.vim-manual|
+		3.1 Health.vim commands ..: |health.vim-commands|
 
-1.1 Commands
+==============================================================================
+2. Health.vim introduction			*health.vim-intro*
+
+Debugging common issues is a time consuming task that many developers would
+like to eliminate, and where elimination is impossible, minimize. Many common
+questions and difficulties could be answered by a simple check of an
+environment variable or a setting that the user has made. However, even with
+FAQs and other manuals, it can be difficult to suggest the path a user should
+take without knowing some information about their system.
+
+Health.vim aims to solve this problem in two ways for both core and plugin
+maintainers.
+
+The way this is done is to provide an interface for that users will know to
+check first before posting questsion in the issue tracker, dev list, etc. This
+is similar to how |:help| functions currently. The user experiencing
+difficulty can run |:CheckHealth| to view the status of her system.
```

Comment:
A pretty common style is to alternate male/female pronouns across paragraphs.  However, the usage of "her" here does standout since most of the other text uses terminology like "the user".  It'd be good to be consistent one way or the other.